fv3_cap.F90

```diff
@@ -872,12 +889,11 @@ subroutine ModelAdvance(gcomp, rc)
     ! local variables
     type(ESMF_Clock)            :: clock
     type(ESMF_Time)             :: currTime, startTime, stopTime
-    type(ESMF_TimeInterval)     :: timeStep
+    ! type(ESMF_TimeInterval)     :: timeStep
 
-    integer                     :: i, urc
     character(len=*),parameter  :: subname='(fv3_cap:ModelAdvance)'
     character(240)              :: msgString
-    character(240)              :: startTime_str, currTime_str, stopTime_str, timeStep_str
+    ! character(240)              :: startTime_str, currTime_str, stopTime_str, timeStep_str
```

Comment:
I think in ModelAdvance, the integrate loop should be removed, we just need the code inside the loop.

Reply:
Let me try that.

Reply:
`cpld_debug_p7` test passed, but `control` failed.

Reply:
Thanks for testing